MiguelAngelVidalLiborio/Dia4-26Agosto/HackerRank/JavaOutPutFormatting.java

```diff
@@ -0,0 +1,23 @@
+package hackerrank;
+
+import java.util.Scanner;
+
+public class JavaOutPutFormatting {
+
+	public static final Scanner scanner = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int limitLoop=3;
+		
+		System.out.println("please enter your programming language and how much you love it");
+		System.out.println("================================");
```

Comment:
Avoid the usage of magic strings

Magic Strings
https://deviq.com/antipatterns/magic-strings